src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java

```diff
@@ -263,32 +267,37 @@ public final void setAllowSamelineMultipleAnnotations(boolean allow) {
     @Override
     public void visitToken(DetailAST ast) {
         final DetailAST modifiersNode = ast.findFirstToken(TokenTypes.MODIFIERS);
-
         if (modifiersNode != null) {
-            checkAnnotations(modifiersNode, getExpectedAnnotationIndentation(modifiersNode));
+            checkAnnotations(modifiersNode);
+        }
+        else {
+            final DetailAST annotationsNode = ast.findFirstToken(TokenTypes.ANNOTATIONS);
+            if (annotationsNode != null) {
+                checkAnnotations(annotationsNode);
+            }
         }
     }
 
     /**
      * Returns an expected annotation indentation.
      * The expected indentation should be the same as the indentation of the node
      * which is the parent of the target modifier node.
-     * @param modifierNode modifier node.
+     * @param annotationsNode modifiers or annotations node.
```

Comment:
This change doesn't make sense. It is modifiers or annotation, yet previous name or new name just picks 1 side.
Can we just rename this to `node`?